library/std/src/sys/mod.rs

```diff
@@ -76,3 +76,52 @@ cfg_if::cfg_if! {
         pub mod c;
     }
 }
+
+#[cfg(not(test))]
```

Comment:
What's up with this cfg? It appears that there will be no impl of these functions under cfg test.

Reply:
:shrug: the code in std is also gated with this.